documentation/book/ref-kafka-bridge-http-configuration.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-consumer-configuration-{context}']
+= Kafka Bridge Consumer configuration
```

Comment:
Change title: Kafka Bridge HTTP configuration

Reply:
Please use suggestions to improve the process of getting this PR closed ASAP.

Reply:
Updated